### ISSUE TITLE
MAJOR ISSUES:  New Labs are Mixed with the Old Versions!

### ISSUE THREAD

**Issue by tylerfarmer1** [MicrosoftLearning/PL-900-Microsoft-Power-Platform-Fundamentals#105]
The scenario in PL-900 used to be where we are building a solution to track visitors on our campus.  It appears that there is a new solution where we are tracking expenses; however, a lot of the instructions are still referencing the visitor solution!

Some examples:

# Lab 1: Data Modeling

In the "Scenario" it explains how we're going to track expenses.  But then R1, R2, and R3 are about scheduling and tracking campus visits.

# Lab 2: Canvas App

Again, the "Scenario" is about Expenses but the "High-level lab steps" are all referencing the Visit table.

# Lab 3: Model-driven App

This one is much more serious.   
## Exercise 1: Customize Views and Form

Task 1 is fine, to edit the Expense Report Form.   However, Task 2 is to "Edit the Active Visits View" which is not part of this Expense Report solution...The Visit table hasn't even been created.   This is left over from the previous version of this course.   The actual step-by-step instructions (thankfully) only reference the Expense Report views, but anything that references "today's visits" is left over from the previous version of the course.

Lab 4 has similar issues.

Lab 5 - for some strange reason - is still based on the "visitor" solution.   If we're going to use the "expenses" solution through most of the class, then the Power BI lab should also be changed to support that scenario.

Basically, search for the word "visit" in any of the labs.  If you see that word, then there is a problem.   

Or...my opinion...revert this the PL-900 back to the Visits scenario... It worked just fine the last 6 times I taught it.

**Comment by valinciab** [comment]
I agree.  I delivered the course back in mid  March and noticed the discrepancies and was hoping they would have been corrected before the next course (this week) but I see its still showing visits information in the  lab instructions that should reference the new Expenses scenario data.

**Comment by fonsell** [comment]
And the problem still remains - it's already 6th May. Is anyone planning to fix this? ;(

**Comment by feridunkadir** [comment]
Any comment from Microsoft?  What happened to quality checking changes?

**Comment by CeleneA** [comment]
Apologies for the delay. The issue has been addressed.

**Comment by feridunkadir** [comment]
Wonderful, thank you!

**Comment by feridunkadir** [comment]
Just noticed that Lab 5 still references visits. Is that intended?
And in Lab 1, data modelling, the data modelling document shows visits and buildings with no reference to expenses.
Also, in Lab 1, the steps have students create an Expenses table but later labs refer to an Expense Report table.
